lib/ui/string.rb

```diff
@@ -0,0 +1,25 @@
+class String
+  def colorize(color_code)
+    "\e[#{color_code}m#{self}\e[0m"
+  end
+
+  def dim
+    "\e[2m#{self}\e[22m"
+  end
+
+  def green
+    colorize(32)
+  end
+
+  def magenta
+    colorize(35)
+  end
+
+  def blue
+    colorize(34)
+  end
+
+  def yellow
+    colorize(33)
+  end
+end
```

Comment:
ruby makes it super easy to extend any existing object, and this is a very ruby way to solve this problem, but I'd like you to solve this using some sort of presentation layer. It's a little jarring to see `'foo'.blue`, it doesn't fit my worldview of a string literal.

Reply:
Made it into a [colorizer class](https://github.com/saralein/tic-tac-toe_ruby/pull/18/files#diff-025f4e3018e7f6d7253cde553a91e93bR1).